DBZ-7786 Add MariaDB connector documentation (#5582)
debezium/debezium: documentation/modules/ROOT/partials/modules/all-connectors/ref-connector-monitoring-streaming-metrics.adoc

|The current volume, in bytes, of records in the queue.

|===

ifdef::MARIADB,MYSQL[]
The {prodname} {connector-name} connector also provides the following additional streaming metrics:

.Descriptions of additional {connector-name} streaming metrics
[cols="3,2,5",options="header"]
|===
|Attribute |Type |Description

|[[binlog-filename]]<<binlog-filename,`+BinlogFilename+`>>
|`string`
|The name of the binlog file that the connector has most recently read.

|[[binlog-position]]<<binlog-position,`+BinlogPosition+`>>
|`long`
|The most recent position (in bytes) within the binlog that the connector has read.

|[[is-gtid-mode-enabled]]<<is-gtid-mode-enabled,`+IsGtidModeEnabled+`>>
|`boolean`

> |Flag that denotes whether the connector is currently tracking GTIDs from MariaDB server.

Author: ```suggestion
|Flag that denotes whether the connector is currently tracking GTIDs from {connector-name} server.
```

|[[gtid-set]]<<gtid-set,`+GtidSet+`>>
|`string`
|The string representation of the most recent GTID set processed by the connector when reading the binlog.

|[[number-of-skipped-events]]<<number-of-skipped-events,`+NumberOfSkippedEvents+`>>
|`long`

> |The number of events that have been skipped by the MariaDB connector. Typically events are skipped due to a malformed or unparseable event from MariaDB's binlog.

Author: ```suggestion
|The number of events that have been skipped by the {connector-name} connector. Typically events are skipped due to a malformed or unparseable event from {connector-name}'s binlog.
```

|[[number-of-disconnects]]<<number-of-disconnects,`+NumberOfDisconnects+`>>
|`long`

> |The number of disconnects by the MariaDB connector.

Author: ```suggestion
|The number of disconnects by the {connector-name} connector.
```
